netbox/dcim/tables/devices.py

```diff
@@ -557,6 +557,12 @@ class BaseInterfaceTable(NetBoxTable):
         orderable=False,
         verbose_name=_('IP Addresses')
     )
+    virtual_ips = tables.TemplateColumn(
```

Comment:
This can be a `ManyToManyColumn` with `linkify_item=True` IMO. We should avoid custom template code where possible.